BREAKING.md

```diff
@@ -188,6 +194,14 @@ The `is` method is removed from SharedObject. This was being used to detect Shar
 The `id` property of IContainerContext was deprecated and now removed. The `id` property of ContainerContext was deprecated and now removed. id should not be exposed at
 runtime level anymore. Instead, get from container's resolvedURL if necessary.
 
+### Remove raiseContainerWarning property
+
+`raiseContainerWarning` property is removed from the following interfaces in release 0.56: `IContainerRuntime`, `IFluidDataStoreRuntime`, `IFluidDataStoreContext`, `IContainerRuntime`, `IFluidDataStoreRuntime`. It was deprecated in `IContainerContext` and it usage would be completely removed. The advised migration step for partners is to generate their own telemetry/logging events.
+
+### Set logger property as optional parameter in IContainerContext
+
+The `logger` property from `IContainerContext` is set as an optional paramter and would be removed in the next release. Use `taggedLogger` instead. Loggers passed to `ContainerContext` will need to support tagged events.
```

Comment:
```suggestion
The `logger` property from `IContainerContext` is now optional. It will be removed completely in a future release. Use `taggedLogger` instead. Loggers passed to `ContainerContext` will need to support tagged events.
```